### PR TITLE
Rectify login page styling

### LINKED ISSUE #1183: Login Page Unresponsive
### What happened just before the problem occurred
Here's The Login Page:

image

As you can see its pretty unresponsive, It would be better if it was center aligned with some padding

### PR DESCRIPTION
**Pull request by gr455**
Fixes #1183 (<=== Add issue number here)
Adds padding and allignment for contents on the login page
Screenshot from 2020-02-05 18-35-18

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/assets/stylesheets/style.scss [hunk 1]
```diff
@@ -318,6 +318,11 @@ label small {
   }
 }
 
+.container{
```
Comment (alaxalves) [inline app/assets/stylesheets/style.scss]: modifying the entire `container` class might affect other parts of MK, doesn't it? Couldn't you use ids or use the previously defined class?

Comment (gr455) [reply app/assets/stylesheets/style.scss]: I used an id. Thanks for the suggestion

**Comment by gr455** [comment]
@publiclab/mapknitter-reviewers Why is Travis failing? Can someone help me

**Comment by cesswairimu** [comment]
Restartinf travis

**Comment by cesswairimu** [comment]
@alaxalves could you please take a look again...need your approval before we are able to merge this.Thanks

**Comment by alaxalves** [comment]
LGTM!!

**Pull request merged**